### PR TITLE
Document Taupage erase_on_boot flag

### PR DESCRIPTION
**Pull request by slitsche**
Document the changed behavior for **erase_on_boot** flag of Taupage.

### CODE REVIEW

File: components/taupage.rst [hunk 1]
```diff
@@ -400,6 +400,28 @@ IAM-Role::
        ]
      }
 
+.. _iamEraseOnBootTag:
+
+IAM-Role for use with **Taupage:erase-on-boot** tag on the EBS volume::
```
Comment (Raffo) [inline components/taupage.rst]: there's a double `:`.

Comment (hjacobs) [reply components/taupage.rst]: @Raffo that's valid RST format, but it should probably changed to `..code-block:: yaml`

Comment (Raffo) [reply components/taupage.rst]: Ah, misread it.

**Comment by a1exsh** [comment]
:+1:

**Comment by Raffo** [comment]
👍

**Pull request merged**